### PR TITLE
Implement correx ch1-6 and skills 1C in Maternal Care

### PR DESCRIPTION
**Pull request by LaurenEllwood**

### CODE REVIEW

File: maternal-care/text/01.md [hunk 1]
```diff
@@ -562,79 +564,79 @@ A waiting area (obstetric village), where cheap accommodation is available for p
...
 There are many advantages to a hand-held Maternity Case Record that includes an antenatal card which records all the patient's antenatal information. It is simple, cheap, and effective. It is uncommon for patients to lose their records. The clinical record is then always available wherever the patient presents for care. The clinic need only record the patient's personal details such as name, address and age together with the dates of her clinic visits and the result of any special investigations.
 
-On the one side of the card are recorded the patient's personal details, history, estimated gestational age, examination findings, results of the special investigations, plan of management, and proposed future family planning. On the other side are recorded all the maternal and fetal observations made during pregnancy.
+One page of the Maternity Case Records is for the patient’s personal details, history, estimated gestational age, examination findings, results of the special investigations, plan of management, and proposed future family planning. The next page that contain the symphysis-fundus height growth curve are for all the maternal and fetal observations made during pregnancy.
```
Comment (KlaraSkinner) [inline maternal-care/text/01.md]: "The next page that contain**s** the symphysis-fundus height growth curve **is** for..."?

File: maternal-care/text/01.md [hunk 2]
```diff
@@ -701,71 +709,71 @@ The severity and progression of HIV infection during pregnancy can be monitored
...
 
-> All HIV-positive women must be commenced on HAART.
+> All HIV-positive women must be commenced on antiretroviral treatment.
 
-### 1-65 What antiretroviral (ARV) treatment is given during pregnancy?
+### 1-66 What antiretroviral treatment is given during pregnancy?
 
-ARV treatment aims at improving the mother's health and reducing the risk of the mother infecting her fetus and newborn infant with HIV (prevention of mother-to-child transmission or PMTCT). ARV treatment should be started at 14 weeks or as soon as possible thereafter and will continue as lifelong treatment. It will reduce the risk of HIV transmission from mother to infant to less than 2%, compared to 30% when ARVs are not taken.
+Antiretroviral treatment (ARV treatment) aims at improving the mother’s health and reducing the risk of the mother infecting her fetus and newborn infant with HIV (prevention of mother-to-child transmission or PMTCT). ARV treatment should be started at 7 weeks or as soon as possible thereafter and will continue as lifelong treatment. It will reduce the risk of HIV transmission from mother to infant to less than 2%, compared to 30% when ARVs are not taken.
```
Comment (KlaraSkinner) [inline maternal-care/text/01.md]: On line 714, and in _Adult HIV_, Antiretroviral treatment is abbreviated as ART, not ARV treatment. Shall we amend here, or does this open up a can of worms for consistency of use throughout?

File: maternal-care/text/01.md [hunk 3]
```diff
@@ -701,71 +709,71 @@ The severity and progression of HIV infection during pregnancy can be monitored
...
+:	WHO option B+, rolled out in 2012 recommended a fixed drug combination (FDC) to be used as a once a day single tablet. FDC consist of tenofovir, emtricitabine and efavirenz. In 2019 FDC was replaced by a new fixed drug combination (TLD) consist of tenofovir, lamivudine and dolutegravir.
 
+### 1-67 What is antiretroviral treatment?
 
-### 1-66 What is antiretroviral treatment?
+The aim of ARV treatment is to lower the viral load and allow the immune system to recover. This will both reduce the risk of HIV transmission to the infant and to treat the mother’s HIV infection. ARV treatment of choice consists of taking tenofovir (TDF) 300 mg, lamivudine (3TC) 300 mg and dolutegravir (DTG) 50 mg as a single fixed dose combination (TLD) tablet daily.
```
Comment (KlaraSkinner) [inline maternal-care/text/01.md]: ART vs ARV treatment may be something we need to query with Dave and Gerhard, looking at what follows here...

File: maternal-care/text/03.md [hunk 4]
```diff
@@ -189,38 +181,48 @@ The severity of pre-eclampsia can be graded by:
...
+	    *   An alanine aminotransferase and/or aspartate aminotransferase of 40 iu/l or more indication liver disease
+	    *   A platelet count of less than 100,000/microliter 
+3.  *Eclampsia*: Eclampsia is diagnosed when a patient with any of the grades of pre-eclampsia has a convulsion. 
 
 > If there is any doubt about the grade of pre-eclampsia, the patient should always be placed in the more severe grade.
 
 Patients who improve on bed rest should be kept in the grade of pre-eclampsia which they were given at the initial evaluation on admission. Further management should be in accordance with this grade.
 
-### 3-17 What are the symptoms and signs of imminent eclampsia?
+With the new terminology imminent eclampsia is now called pre-eclampsia with severe features or pre-eclampsia with features of severe disease. The latter is used in this chapter
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: Missing full stop

File: maternal-care/text/03.md [hunk 5]
```diff
@@ -240,17 +242,17 @@ In the Western Cape, the incidence of eclampsia is 1 per 1000 pregnancies.
...
 
-1.	A rise in diastolic and/or systolic blood pressure.
-2.	Proteinuria.
-3.	Symptoms and signs of imminent eclampsia.
+1.	A rise in blood pressure. 
+2.	Proteinuria. 
+3.	Symptoms and signs of severe disease. 
 
-Patients with an obstetric history of pre-eclampsia that developed late in the second trimester or early in the third trimester, must receive 75 mg aspirin (a quarter Disprin) daily from a gestational age of 14 weeks. This will reduce the risk that pre-eclampsia may develop.
+Patients with an obstetric history of pre-eclampsia and chronic hypertension, must receive 150 mg aspirin (a half Disprin) daily from a gestational age of 14 weeks. Calcium 1 gram daily is also recommended (two 500 mg tablets daily). The Calcium tablets need to be taken 4 hours apart from the iron tablets as Calcium reduce the absorption of iron. These measures will reduce the risk that pre-eclampsia will develop.
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: "Calcium reduce**s**"?

File: maternal-care/text/03.md [hunk 6]
```diff
@@ -331,38 +337,38 @@ The steps in the management of severe pre-eclampsia are:
...
+*	If dihydralazine was used, an ampoule of dihydralazine (25 mg) should be mixed with 10 ml of sterile water. Bolus doses of 2 ml (5 mg) are given slowly intravenously, at 30-minute intervals, until the diastolic blood pressure drops to below 110 mm Hg and/or the systolic blood pressure below 160 mm Hg.
 
 Note
 :	If dihydralazine is not available, labetalol (Trandate) 20 mg is given intravenously. If a decline in blood pressure does not occur after 10 minutes a further 40 mg is given intravenously.
 
 *Step 4*: When the blood pressure is controlled, the patient is transferred to a level 2 or 3 hospital.
 
-> Patients with severe pre-eclampsia or imminent eclampsia must always be stabilised before they are transferred, or until further management is decided upon.
+> Patients with pre-eclampsia and features of severe diseasemust always be stabilised before they are transferred, or until further management is decided upon.
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: Space between 'disease' and 'must'

File: maternal-care/text/03.md [hunk 7]
```diff
@@ -331,38 +337,38 @@ The steps in the management of severe pre-eclampsia are:
...
 
 ### 3-30 What can be done to ensure maximal safety for the patient during her transfer to hospital?
 
-1.	A doctor or registered nurse or midwife should accompany the patient.
+1.	Trained ambulance personnel and if not available a doctor or registered nurse should accompany the patient.
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: V confusing language usage here :(

File: maternal-care/text/03.md [hunk 8]
```diff
@@ -546,35 +550,35 @@ She should be admitted to hospital as all patients with 2+ proteinuria must be h
 
 ### 2. On examining this patient you observe that she has increased patellar reflexes i.e. brisk knee jerks. How should this observation alter her management?
 
-Increased tendon reflexes are a sign of imminent eclampsia. The diagnosis must be made, irrespective of the degree of hypertension or the amount of proteinuria. To prevent the development of eclampsia, the patient must be given magnesium sulphate.
+Increased tendon reflexes are a sign of pre-eclampsia with severe features. The diagnosis must be made, irrespective of the degree of hypertension or the amount of proteinuria. To prevent the development of eclampsia, the patient must be given magnesium sulphate.
 
 ### 3. What is the danger to this patient’s health?
 
-The patient has severe pre-eclampsia. Therefore, the immediate danger to her life is the development of eclampsia or an intracerebral haemorrhage.
+The patient has pre-eclampsia with severefeatures. Therefore, the immediate danger to her life is the development of eclampsia, an intracerebral haemorrhage or pulmonary oedema.
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: Space between 'severe' and 'features'

File: maternal-care/text/03.md [hunk 9]
```diff
@@ -546,35 +550,35 @@ She should be admitted to hospital as all patients with 2+ proteinuria must be h
...
+No. Sometimes in pre-eclampsia with  severe features, the blood pressure will drop to below 160/110 mm Hg after the loading dose of magnesium sulphate has been given. In that case, no further management is needed for the hypertension. However, if the patient’s blood pressure does not drop after administering the magnesium sulphate, 10 mg (1 capsule) oral nifedipine (Adalat) or intramuscular dihydralazine (Nepresol) 6.25 mg should be given.
 
 ## Case study 3
 
 While working at a level 1 hospital you admit a patient with a blood pressure of 170/120 mm Hg and 3+ proteinuria. She is 32 weeks pregnant. On further questioning and examination, she has no symptoms or signs of imminent eclampsia.
 
 ### 1. What is the danger to this patient’s health?
 
-The patient has severe pre-eclampsia. Therefore, the immediate danger to her life is the development of eclampsia or an intracerebral haemorrhage.
+The patient has pre-eclampsia with severefeatures. Therefore, the immediate danger to her life is the development of eclampsia, an intracerebral haemorrhage or pulmonary oedema.
```
Comment (KlaraSkinner) [inline maternal-care/text/03.md]: Space between 'severe' and 'features'

File: maternal-care/text/04.md [hunk 10]
```diff
@@ -418,14 +418,14 @@ Note
...
 5.	A patient with a grade 3 or 4 placenta praevia and a viable fetus of less than 34 weeks, who is going to be managed conservatively, should be managed in at least a level 2 hospital with a neonatal intensive care unit, or a level 3 hospital.
 {:.allow-break}
 
 ### 4-44 When you refer a patient, what precautions should you take to ensure the safety of the patient in transit?
 
-1.	A shocked patient should have 2 intravenous infusion lines with Plasmalyte B or Ringer’s lactate running in fast. A doctor should accompany the patient if possible. If not possible, a registered nurse should accompany her.
-2.	A patient who is no longer bleeding should also have an intravenous infusion and be accompanied by a registered nurse whenever possible.
+1.	A shocked patient should have 2 intravenous infusion lines with Plasmalyte B or Ringer’s lactate running in fast. Trained ambulance personnel and if not available a doctor should accompany the patient if possible. If not possible, a registered nurse should accompany her.
```
Comment (KlaraSkinner) [inline maternal-care/text/04.md]: Confusing language again. We may need to add some commas or reword this instance, the previous one and the one which follows.

File: maternal-care/text/01.md (hunk 2 above)
Comment (LaurenEllwood) [reply maternal-care/text/01.md]: I amended here and did a quick search of the text to keep it consistent. There were only two other instances that needed to change.

Comment (LaurenEllwood) [reply maternal-care/text/01.md]: Nevermind, after seeing your next comment I've decided to leave as is.

File: maternal-care/text/01.md (hunk 3 above)
Comment (LaurenEllwood) [reply maternal-care/text/01.md]: I agree that this should be queried with Dave and Gerhard, things are getting a bit confusing now...

Comment (LaurenEllwood) [reply maternal-care/text/01.md]: Changed my mind again on this. I did a search of the text and found that ART was only used once throughout the book so decided to standardise as ARV treatment and not ART.

File: maternal-care/text/01.md [hunk 11]
```diff
@@ -59,18 +59,18 @@ The aims of good antenatal care are to ensure that pregnancy causes no harm to t
 4. The fetal condition must be repeatedly assessed.
 5. Healthcare education must be provided.
 
-All information relating to the pregnancy must be entered in a patient-held Maternity Case Record. The Maternity Case Record can also serve as a referral letter if a patient is referred to the next level of care and therefore serves as a link between the different levels of care as well as the antenatal clinic and labour ward.
+All information relating to the pregnancy must be entered in a patient-held Maternity Case Records. The antenatal record can also serve as a referral letter if a patient is referred to the next level of care and therefore serves as a link between the different levels of care as well as the antenatal clinic and labour ward.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Maternity Case record instead of "Records"

File: maternal-care/text/01.md [hunk 12]
```diff
@@ -338,24 +343,24 @@ A number of antibody antigen rapid tests are available to diagnose HIV infection
 	2.	The great benefit of the rapid test is that it can be done on site to give same day results.
 
 Note
-:	Ultrasound screening at 11 to 13 weeks for nuchal thickness, or the triple test, is very useful in screening for Down syndrome and other chromosomal abnormalities.
+:	Ultrasound screening at 11 to 13 weeks for nuchal thickness, or the serum triple test, is very useful in screening for Down syndrome and other chromosomal abnormalities.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Left out - "Written informed consent for HIV testing is not a legal requirement in South Africa."

File: maternal-care/text/01.md [hunk 13]
```diff
@@ -370,20 +375,20 @@ Once the patient has been booked for antenatal care, it must be assessed whether
 
 ### 1-32 If no risk factors are found at the booking visit, when should the patient be seen again?
```
Comment (catexedwards) [inline maternal-care/text/01.md]: 1-31 - left out an edit - should must change to could

File: maternal-care/text/01.md [hunk 14]
```diff
@@ -370,20 +375,20 @@ Once the patient has been booked for antenatal care, it must be assessed whether
...
 2.	A patient with a risk factor is followed up sooner if necessary:
 	*	The management of a patient with chronic hypertension would be planned and the patient would be seen a week later.
-	*	A newly diagnosed HIV-positive patient must be seen a week later to obtain her serum creatinine and CD4 count results.
+	*	A newly diagnosed HIV-infected patient must be seen one week later to check her CD4 count and serum creatinine value.
 
 ### 1-34 How should you list risk factors?
 
-All risk factors must be entered on the problem list on the back of the antenatal card. The gestational age when management is needed should be entered opposite the gestational age at the top of the card, e.g. vaginal examination must be done at each visit from 26 to 32 weeks if there is a risk of preterm labour.
+All risk factors must be entered on the problem list on the symphysis-fundus height graph and follow-up visits of the antenatal record. The management needed should be entered opposite the gestational age at the top of the antenatal record, e.g. a glucose profile or umbilical artery Doppler needs to done at 26 weeks if indicated.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Sentence change suggestion: a glucose profile or umbilical artery Doppler needs to _(add "be")_ done at 26 weeks if indicated.

File: maternal-care/text/01.md [hunk 15]
```diff
@@ -520,39 +525,36 @@ A high-risk patient has a problem which requires continuous additional care. For
...
-1.	Every 8 weeks until 28 weeks.
-2.	The next visit is six weeks later at 34 weeks.
-3.	Primigravidas are then seen every 2 weeks from 36 weeks and multigravidas from 38 weeks. However, multigravidas are also seen again at 36 weeks if a breech presentation was found at their 34 week visit.
-4.	Thereafter primigravidas are seen at 40 weeks and 41 weeks while multigravidas are only seen at 41 weeks, if they have not yet delivered.
-
-In some rural areas it may be necessary to see low-risk patients less often because of the large distances involved. The risk of complications with less frequent visits in these patients is minimal. Visits may be scheduled as follows: after the first visit (combining the booking and second visit), the follow-up visits at 28, 34 and 41 weeks.
+1.	Every 6 weeks until 26 weeks.
+2.	The next 2 visits are 4 weeks apart at 30 and 34 weeks. 
+3.	Thereafter every 2 weeks to 40 weeks (i.e. 36, 38 and 40 weeks). 
+4.	At 40 weeks a date for induction of labour will be given if spontaneous onset of labour has not occurred by 41 completed weeks (when the 31st week has been completed i.e.ie 287 days of gestation).
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Is the double i.e correct?

Comment (catexedwards) [inline maternal-care/text/01.md]: Is the double i.e correct?

File: maternal-care/text/01.md [hunk 16]
```diff
@@ -520,39 +525,36 @@ A high-risk patient has a problem which requires continuous additional care. For
...
 2.	Early signs of pre-eclampsia may now be present for the first time, as it is a problem which develops in the second half of pregnancy. Therefore, the patient must be assessed for proteinuria and a rise in the blood pressure.
 3.	Cervical changes in a patient who is at high risk for preterm labour, e.g. a patient with multiple pregnancy, a history of previous preterm labour, or polyhydramnios.
 4.	If the symphysis-fundus height measurement is below the 10th centile, assess the patient for causes of poor fundal growth.
 5.	If the symphysis-fundus height measurement is above the 90th centile, assess the patient for the causes of a uterus larger than dates.
 6.	Anaemia may be detected for the first time during pregnancy.
-7.	Diabetes in pregnancy may present now with glycosuria. If so, a random blood glucose concentration must be measured.
+7.	A glucose profile needs to done on all women at risk for gestational diabetes e.g. age 40 years or more, a BMI of 40 or more, previous gestational diabetes, previous big babies (above4.5 kg), previous shoulder dystocia or a previous stillbirth of unknown cause.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Edit suggestion: A glucose profile needs to _(add: be)_ done on all women at risk for gestational diabetes e.g. age 40 years or more, a BMI of 40 or more, previous gestational diabetes, previous big babies (above4.5 kg), previous shoulder dystocia or a previous stillbirth of unknown cause.

File: maternal-care/text/01.md [hunk 17]
```diff
@@ -562,79 +564,79 @@ A waiting area (obstetric village), where cheap accommodation is available for p
...
 
 > It is important that all antenatal women have a hand-held Maternity Case Record.
 
 ### 1-56 What topics should you discuss with patients during the health education sessions?
 
 The following topics must be discussed:
 
-1.	Danger symptoms and signs.
-2.	Dangerous habits, e.g. smoking or drinking alcohol.
+1.	Danger symptoms.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: It should be: Dangerous Symptoms

File: maternal-care/text/01.md [hunk 18]
```diff
@@ -562,79 +564,79 @@ A waiting area (obstetric village), where cheap accommodation is available for p
...
+2.	Dangerous habits, e.g. smoking, drinking alcohol or illicit drug abuse.
 3.	Healthy eating.
 4.	Family planning.
 5.	Breastfeeding.
 6.	Care of the newborn infant.
-7.	The onset of labour and labour itself must also be included when the patient is a primigravida.
-8.	Avoiding HIV infection or getting counselling if HIV positive.
+7.	The onset of labour and labour must also be included when the patient is a primigravida. 
+8.	Avoiding HIV infection or the importance of adherance with antiretrovirals if HIV positive.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Possible spelling change adherance - adherence

File: maternal-care/text/01.md [hunk 19]
```diff
@@ -643,39 +645,45 @@ The following topics must be discussed:
 	*	Folic acid supplementation is required for mothers with a history of previous babies with neural tube defects and women living in endemic malaria areas. One 5 mg tablet should be taken daily from the time pregnancy is planned.
 {:.allow-break}
 
+### 1-58 Which supplements should routinely be taken during pregnancy?
+
+1.	Iron supplements should be routinely taken as explained in sections 13-19 to 13-22.
+2.	Calcium supplementation decrease the risk of developing pre-eclampsia, especially in areas where dietary intake of Calcium is low. Calcium 1 grams daily is recommended (two 500 mg tablets daily). The Calcium tablets need to be taken 4 hours apart from the iron tablets as Calcium reduce the absorption of iron.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Possible edit of grams to gram?

Comment (catexedwards) [reply maternal-care/text/01.md]: Possible edit of reduce to reduces?

File: maternal-care/text/01.md [hunk 20]
```diff
@@ -643,39 +645,45 @@ The following topics must be discussed:
...
+3.	Folic acid 5 mg supplementation is routinely provided.  Mothers with a history of previous babies with neural tube defects should be take folic acid from the time pregnancy is planned.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Possible edit of take to taking?

File: maternal-care/text/01.md [hunk 21]
```diff
@@ -643,39 +645,45 @@ The following topics must be discussed:
...
 
-### 1-59 Is AIDS an important cause of maternal death?
+### 1-60 Is AIDS an important cause of maternal death?
 
 As the HIV epidemic spreads, the number of pregnant women dying of AIDS has increased dramatically. In some countries, such as South Africa, AIDS is now the commonest cause of maternal death. Therefore all pregnant women must be screened for HIV infection.
 
 Note
-:	The Second, Third and Forth Interim Report on Confidential Enquiries into Maternal Deaths in South Africa showed that AIDS was the commonest cause of maternal death.
+:	According the Confidential Enquiries into Maternal Deaths in South Africa’s 5th Saving Mothers Report for the 2014-2016 triennium 40% of maternal deaths were caused by AIDS.  Subsequently there was a decrease in deaths due to the roll out of antiretroviral therapy for all people living with HIV in 2012.  According to the 2017 annual Saving Mothers Report AIDS remains the commonest cause of maternal death, causing 25% of deaths.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Add 'to' after according?

File: maternal-care/text/01.md [hunk 22]
```diff
@@ -701,71 +709,71 @@ The severity and progression of HIV infection during pregnancy can be monitored
 	*	Extrapulmonary tuberculosis (TB).
 {:.allow-break}
 
-### 1-64 How should pregnant women with a positive HIV screening test be managed?
+### 1-65 How should pregnant women with a positive HIV screening test be managed?
 
-It is very important to identify women with HIV infection as soon as possible in pregnancy so that they can be carefully assessed and their management can be planned. HIV positive pregnant women need to be commenced on highly active antiretroviral (HAART) treatment. The HIV management should be integrated into the rest of the antenatal care. All women with a positive HIV screening test must have their CD4 count determined as soon as the HIV screening result is obtained.
+It is very important to identify women with HIV infection at the fist antenatal visit so that they can be carefully assessed and their management can be planned. HIV positive pregnant women need to be commenced on antiretroviral (ARV) treatment. The HIV management should be integrated into the rest of the antenatal care. All women with a positive HIV screening test must have their CD4 count determined as soon as the HIV screening result is obtained.
```
Comment (catexedwards) [inline maternal-care/text/01.md]: Change fist to first

File: maternal-care/text/01.md [hunk 23]
```diff
@@ -777,35 +785,37 @@ Common early side effects during the first few weeks of starting ARV drugs inclu
...
-*	Lactic acidosis is a late but serious side effect, most often caused by stavudine (d4T), but also caused by AZT and TDF. Lactic acidosis presents with weight loss, tiredness, nausea, vomiting, abdominal pain and shortness of breath in patients that have been well and on ARV treatment for months.
+*	EFV may cause mild skin rashes.
+*	AZT suppresses bone marrow and can cause anaemia. There may also be a reduction in the white cell and platelet count.
+*	Hepatitis can be caused by EFV.
+*	Lactic acidosis is a late but serious side effect, especially caused by AZT and TDF. It presents with weight loss, tiredness, nausea, vomiting, abdominal pain and shortness of breath in patients that have been well and on ARV treatment for months.
 
 Staff at primary-care clinics must be aware of and look out for these very important side effects.
 
-### 1-73 Is HIV and TB co-infection common in pregnancy?
+### 1-74 Is HIV and TB co-infection common in pregnancy?
```
Comment (catexedwards) [inline maternal-care/text/01.md]: 76 in the document but the follow on numbering is 74

File: maternal-care/text/02.md [hunk 24]
```diff
@@ -392,7 +399,7 @@ Immediately proceed with fetal resuscitation, as follows:
 It is important that you know how to give fetal resuscitation, as it is a life-saving procedure when fetal distress is present, both during the antepartum period and in labour.
 
 Note
-:	Uterine contractions can also be suppressed if 30 mg nifedipine (Adalat) is given my mouth (1 capsule = 10 mg). The 3 capsules must be swallowed and not used sublingually.
+:	Uterine contractions can also be suppressed if 30 mg nifedipine (Adalat) is given my mouth (1 capsule = 10 mg). The 3 capsules must be swallowed and not used sublingually. Suppression of uterine contractions will only occur after a delay of about 20 minutes followed by a longer period of suppression that is useful if the patient requires transfer.
```
Comment (catexedwards) [inline maternal-care/text/02.md]: Is it perhaps given by mouth instead of my mouth?

File: maternal-care/text/03.md [hunk 25]
```diff
@@ -299,29 +301,33 @@ Patients with a gestation of less than 36 weeks must be managed as described in
 
 ### 3-27 What special investigations are indicated in pre-eclampsia?
 
-1.	An MSU must be sent to the laboratory for culture, as a urinary tract infection may be responsible for the proteinuria.
-2.	A platelet count must be done, if a laboratory is available. A platelet count of less than 100 000 is an indication for referral of the patient to a level 2 hospital.
+1.	An MSU must be examined microscopically for a urinary tract infection, or sent to the laboratory for culture, as a urinary tract infection may be responsible for the proteinuria. 
+2.	A platelet count must be done, if a laboratory is available. A platelet count of less than 100 000 is an indication for referral of the patient to a level 2 hospital. 
+3.	A serum creatinine to assess renal function.
+4.	An ultrasound examination to estimate the gestational age and fetal weight.
+5.	A umbilical artery Doppler to asses placental function.
```
Comment (catexedwards) [inline maternal-care/text/03.md]: Change "asses" to "assess"

File: maternal-care/text/03.md [hunk 26]
```diff
@@ -331,38 +337,38 @@ The steps in the management of severe pre-eclampsia are:
...
 *	If the diastolic blood pressure remains 110 mm Hg and/or the systolic blood pressure 160 mm Hg or more after 30 minutes, patients who received 10 mg nifedipine orally can be given a second dose of 10 mg nifedipine orally. If necessary, 10 mg of nifedipine can be repeated half-hourly up to a maximum dose of 50 mg.
 *	Nifedipine 10 mg capsules must always be given orally and not given sublingually (under the tongue). The 10 mg capsules must not be confused with Adalat XL tablets which are slowly dissolved and not suitable for rapidly lowering blood pressure.
 
 Or
 
-*	If dihydralazine was used, an ampoule of dihydralazine (25 mg) should be mixed with 20 ml of sterile water. Bolus doses of 2 ml (2.5 mg) are given slowly intravenously, at 20-minute intervals, until the diastolic blood pressure drops to below 110 mm Hg and/or the systolic blood pressure below 160 mm Hg.
+*	If dihydralazine was used, an ampoule of dihydralazine (25 mg) should be mixed with 10 ml of sterile water. Bolus doses of 2 ml (5 mg) are given slowly intravenously, at 30-minute intervals, until the diastolic blood pressure drops to below 110 mm Hg and/or the systolic blood pressure below 160 mm Hg.
```
Comment (catexedwards) [inline maternal-care/text/03.md]: Add sentence at the end: A total of 3 doses of 5 mg each can be given.

File: maternal-care/text/01.md (hunk 23 above)
Comment (LaurenEllwood) [reply maternal-care/text/01.md]: Yes, the source document was numbered incorrectly so am leaving this as 74.

**Pull request merged**